solution/backend/content_search/functions.py

```diff
@@ -130,3 +133,16 @@ def index_group(resources):
     '''
     for res in resources:
         add_to_index(res)
+
+
+def build_lambda_client():
+    if settings.USE_AWS_TOKEN:
```

Comment:
we use the aws token to determine if its local or deployed.  This is how we determine it for s3